boards/arm/sama5/sama5d4-ek/include/nsh_romfsimg.h

```diff
@@ -1,3 +1,23 @@
+/***************************************************************************
+ * boards/arm/sama5/sama5d4-ek/include/nsh_romfsimg.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ***************************************************************************/
+
 unsigned char romfs_img[] = {
```

Comment:
as far as I know such files are autogenerated. I'm not sure if the proper steps to regenerate are run as a part of the build process. Probably not, but the licence information will be lost as soon as someone runs a script to generate a new content.

Reply:
we can ask that the license is added to the contribution when the file is regenerated